fuse_constraints/include/fuse_constraints/absolute_pose_2d_stamped_constraint.h

```diff
@@ -63,7 +63,7 @@ namespace fuse_constraints
 class AbsolutePose2DStampedConstraint : public fuse_core::Constraint
 {
 public:
-  SMART_PTR_DEFINITIONS(AbsolutePose2DStampedConstraint);
+  FUSE_CONSTRAINT_DEFINITIONS_WITH_EIGEN(AbsolutePose2DStampedConstraint);
```

Comment:
I think this one doesn't need to be aligned, so you should do `FUSE_CONSTRAINT_DEFINITIONS(AbsolutePose2DStampedConstraint);`

The `Eigen::Vector3d` attribute in https://github.com/locusrobotics/fuse/pull/65/files#diff-bd3a017d88c898ee2cdac33c00c10c0dR145 is not a fixed-size vectorizable Eigen object: https://eigen.tuxfamily.org/dox/group__TopicFixedSizeVectorizable.html

Reply:
Didn't pick up on the importance of the "vectorable" keyword before. Thanks.